dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-3.0/com.ibm.websphere.appserver.restConnector-3.0.feature

```diff
@@ -0,0 +1,27 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.restConnector-3.0
+visibility=public
+singleton=true
+IBM-API-Package: com.ibm.websphere.jmx.connector.rest; type="ibm-api", \
+ com.ibm.ws.jmx.connector.client.rest; type="ibm-api", \
+ com.ibm.websphere.filetransfer; type="ibm-api"
+IBM-ShortName: restConnector-3.0
```

Comment:
looks like this is pretty much the same as `restConnector-2.0`. As discussed in the pod, lets just drop this feature and have everything key off of `restConnector-2.0` instead